### PR TITLE
Updated README 

### PR DESCRIPTION
**Pull request by caref2**
- Updated the README (i hate typing so here is an image)
image

- added "images" folder for preview images

### CODE REVIEW

File: images/j
Comment (elibroftw) [inline images/j:1]: This supposed to be here?

File: Resources/Chrome Matte Black.png
Comment (elibroftw) [inline Resources/Chrome Matte Black.png:1]: Would be good to add in the README (under Note for Myself) how to generate the chrome and last two firefox previews. Did you just do it on your own or use something? If the latter, no need to modify the README.

**Review by elibroftw — CHANGES_REQUESTED** [review]

**Comment by elibroftw** [comment]
Thanks!

**Comment by caref2** [comment]
no problem

**Pull request merged**